docs/guide/circuit_generation.md

```diff
@@ -0,0 +1,292 @@
+    noise_model=NoiseParameters(),
+)
+
+deltakit_czswap_qmen = czwap_qpu_instance.compile_circuit(iswap_qmem_circuit)
+deltakit_czswap_qmen.as_stim_circuit().diagram("timeline-svg-html")
+```
+
+## 2. Stability experiment
+
+Memory experiments check how well logical observables are preserved through time.
+[Stability experiments](https://arxiv.org/abs/2204.13834) check how well a quantum error
+correction system can move logical observables through space.
+
+Below you will generate a small stability experiment.
+You will need an API client object to generate a stability experiments.
+
+```{code-cell} ipython3
+from deltakit_explorer.codes import css_code_stability_circuit
+
+deltakit_stability = css_code_stability_circuit(
+    css_code=RotatedPlanarCode(width=3, height=5),  # rectangular patch
+    num_rounds=7,  # time dimension
+    logical_basis=PauliBasis.Z,
+    client=cloud,
+)
+stim_stability = deltakit_stability.as_stim_circuit()
+```
+
+## 3. qLDPC codes
+
+You may use Deltakit to generate QEC experiments, which use qLDPC codes.
+qLDPC codes are interesting to explore, as for the price of connectivity you may achieve higher density of logical qubits.
+The code below will generate a memory experiment using the bivariate bicycle code.
+You will compile it for the ion trap native gate set.
+In this experiment you remove Pauli gates (`X`, `Y`, `Z`, `I`).
+Pauli gate effect on the result of simulations can be tracked classically, so in the
+context of a Stim simulation (or a simple experiment on a real QPU) they may be removed,
+thus making execution faster (without loss of correctness).
+You prefer to keep Pauli gates, if you plan to run the experiment in other simulators or using real QPUs.
+
+```{code-cell} ipython3
+from deltakit_explorer.codes import BivariateBicycleCode
+
+code = BivariateBicycleCode(
+    param_m=6,
+    param_l=6,
+    m_A_powers=[3, 1, 2],
+    m_B_powers=[3, 1, 2],
+)
+textbook_circuit = css_code_memory_circuit(code, num_rounds=3, logical_basis=PauliBasis.Z)
+
+ions_qpu_instance = QPU(
+    qubits=textbook_circuit.qubits,
+    native_gates_and_times=NativeGateSetAndTimes(
+        # times here are 0, as we are not using them for noise definition
+        one_qubit_gates={
+            # support X and Z rotations:
+            gates.X: 0.0, gates.SQRT_X: 0.0, gates.SQRT_X_DAG: 0.0,
+            gates.Z: 0.0, gates.S: 0.0, gates.S_DAG: 0.0,
+        },
+        two_qubit_gates={gates.SQRT_XX: 0.0},
+        reset_gates={gates.RZ: 0.0},
+        measurement_gates={gates.MZ: 0.0},
+    ),
+)
+
+ion_trap_qmem_qldpc = ions_qpu_instance.compile_circuit(
+    textbook_circuit,
+    remove_paulis=True,  # this is to remove Pauli gates from the circuits
+)
+ion_trap_qmem_qldpc.as_stim_circuit().diagram("timeline-3d")
+```
+
+## Creating your own codes
+
+In this document you have learnt how to generate Quantum Memory and Stability experiments with Repetition, Rotated Planar, and qLDPC codes in different native gate sets.
```

Comment:
```suggestion
In this document you have learned how to generate Quantum Memory and Stability experiments with Repetition, Rotated Planar, and qLDPC codes in different native gate sets.
```

Reply:
We've checked, across the docs we stick to British English. We may rewrite into American version, but learnt aligns with the rest of the text better.